### PR TITLE
Read analytics id from both legacy files

### PR DESCRIPTION
**Pull request by fvictorio**

### CODE REVIEW

File: packages/hardhat-core/src/internal/cli/analytics.ts [hunk 1]
```diff
@@ -126,7 +124,7 @@ export class Analytics {
       dp: `/task/${taskName}`,
 
       // Host name.
-      dh: "cli.buidler.dev",
+      dh: "cli.hardhat.dev",
```
Comment (fvictorio) [inline packages/hardhat-core/src/internal/cli/analytics.ts:127]: Not 100% sure about this, but I think it makes sense?

Comment (alcuadrado) [reply packages/hardhat-core/src/internal/cli/analytics.ts:127]: We don't own this domain 😭💔😰

Comment (alcuadrado) [reply packages/hardhat-core/src/internal/cli/analytics.ts:127]:
```suggestion
      dh: "cli.usehardhat.com",
```

**Review by alcuadrado — APPROVED** [review]
We just need to change the URL and some error numbers

**Pull request merged**